frontend/src/app/services/auth.service.ts

```diff
@@ -23,14 +25,8 @@ export class AuthService {
         client_secret: password,
       })
       .pipe(
-        tap({
-          next: (response: any) => {
-            this.handleLoginSuccess(response);
-          },
-          error: (error: any) => {
-            this.handleLoginError(error);
-          },
-        }),
+        tap((response) => this.handleLoginSuccess(response)),
```

Comment:
This feels like a more rxjs'y way of doing the same. At least I think this should work... 😁

I haven't actually set up my backend yet so haven't had a chance to test this out.